### DISCUSSION TITLE
Custom Width and Height

### DISCUSSION THREAD

**Discussion by gigius** [BalintKomjati/smooth-infinite-zoom#7] [Ideas]
Hi, I tried the notebook and it is great (even more so now that Google necessarily requires the paid option to use a webui). I tried the "advanced" notebook and was wondering if it is possible to implement a customization of the height and width of the video you want to generate. I entered a variable for the width but if I enter different values I get an error.

Thanks

**Comment by BalintKomjati** [comment]
Since the network was trained on 512*512 images it does not work well on other resolutions. I do not recall it drops an error but could be. I would not recommend to deviate from this resolution in general as the outcome will not be "exciting" even if it runs w/o an error.

**Comment by BalintKomjati** [comment]
Generating 512*512 images and then outpainting to an other higher non-cubic resolution would be my way to go. But it is not implemented in this repo...
